style.css

```diff
@@ -0,0 +1,468 @@
+    font-size: 13px;
+    font-stretch: 100%;
+    font-weight: 500;
+    letter-spacing: 0.007px;
+    line-height: 17px;
+    text-transform: uppercase;
+    cursor: pointer;
+  }
+  
+  .commnet-head {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+    margin-bottom: 25px;
+    margin-left: 85px;
+  }
+  
+  .comment-tab svg {
+    width: 24px;
+    height: 24px;
+    fill: rgb(144, 144, 144);
+    margin-top: 5px;
+    margin-right: 8px;
+    cursor: pointer;
+  }
+  
+  .comment-tab span {
+    font-size: 14px;
+    font-stretch: 100%;
+    font-weight: 500;
+    line-height: 20px;
+    text-transform: uppercase;
+    color: rgb(96, 96, 96);
+    cursor: pointer;
+  }
+  
+  .comment-tab {
+    display: flex;
+    align-items: center;
+    margin-top: 5px;
+    margin-bottom: 10px;
+  }
+  
+  .comment-tab h2 {
+    margin-right: 32px;
+    font-size: 14px;
+  }
+  
+  .comment-form {
+    display: flex;
+    align-items: flex-start;
+    border: none;
+  }
+  
+  .form {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    margin-right: 16px;
+    cursor: pointer;
+  }
+  
+  .comment-form input[type=text] {
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    padding-bottom: 4px;
+    border-bottom: 1px solid;
+    border-color: rgb(134, 129, 129);
+    padding-right: 550px;
+    background-color: transparent;
+    margin-top: 10px;
+  }
+  
+  .comm-cancel {
+    display: flex;
+    margin-top: 50px;
+    margin-right: 480px;
+    gap: 10px;
+    cursor: pointer;
+    position: absolute;
+    right: 5px;
+  }
+  
+  .commenter {
+    display: flex;
+    margin-bottom: 16px;
+    margin-left: 65px;
+  }
+  
+  .author {
+    width: 40px;
+    height: 40px;
+    margin-top: 20px;
+    margin-left: 20px;
+    border-radius: 50%;
+    cursor: pointer;
+  }
+  
+  .comment-contents {
+    display: flex;
+    flex-direction: column;
+    margin-top: 20px;
+    margin-left: 5px;
+    font-size: 11px;
+    cursor: pointer;
+  }
+  
+  .comment-contents span {
+    font-size: 9px;
+    font-weight: lighter;
+  }
+  
+  .body-p {
+    display: flex;
+    position: relative;
+    right: 155px;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    font-stretch: 100%;
+    margin-top: 60px;
+  }
+  
+  .com-action {
+    display: flex;
+    align-items: center;
+    margin-top: 4px;
+    margin-left: -8px;
+  }
+  
+  .com-action button {
+    display: flex;
+    align-items: flex-start;
+    position: relative;
+    top: 50px;
+    right: 715px;
+    margin-right: 8px;
+    padding: 0;
+    background: transparent;
+    color: #606077;
+    border: none;
+    cursor: pointer;
+  }
+  
+  .sidebar {
+    width: 50%;
+    padding: 24px 24px 0 0;
+    margin-top: 5px;
+  }
+  
+  .side-bar {
+    margin-top: 8px;
+  }
+  
+  .side-bar:first-child {
+    margin-top: 0;
+    margin-bottom: 8px;
+  }
+  
+  .side-bar a {
+    display: flex;
+    text-decoration: none;
+  }
```

Comment:
Please add the side video position to the left side video player and margins to your design and try to stick as much as possible.
Kindly add image preview to your left side position of videos.
![159272796-501a4152-15c7-43a5-820f-6ac52904c051](https://user-images.githubusercontent.com/97598663/185981474-bcbe60ed-565d-4b64-a47c-ba082a372568.jpeg)